planet/cli/data.py

```diff
@@ -142,7 +142,9 @@ async def search_create(ctx, name, item_types, filter, daily_email, pretty):
 @pretty
 @click.argument('search_id')
 async def search_get(ctx, search_id, pretty):
-    """ This function obtsains an existing saved search, using the search_id.
+    """ Get saved search.
```

Comment:
This is only a nit, but there is no extra whitespace needed before the first word. In fact, I think this could make `search-get --help` look strangely formatted if click didn't strip the leading space.

```suggestion
    """Get saved search.
```